functions/influxdb+grafana.bash

```diff
@@ -258,18 +258,17 @@ grafana_install(){
   fi
 
   echo -n "$(timestamp) [openHABian] Setting up Grafana service... "
-  # Workaround for strange behavior in CI
-  if ! cond_redirect mkdir -p /run/grafana; then echo "FAILED (mkdir)"; return 1; fi
-  if ! cond_redirect chmod -R 0750 /run/grafana; then echo "FAILED (chmod)"; return 1; fi
-  if ! cond_redirect chown -R grafana:grafana /run/grafana; then echo "FAILED (chown)"; return 1; fi
-  if ! zram_dependency install grafana-server; then return 1; fi
-  if zram_is_installed && ! mkdir -p /opt/zram/log.bind/grafana /var/log/grafana && chown grafana /opt/zram/log.bind/grafana /var/log/grafana; then echo "FAILED (create zram logdir)"; return 1; fi
-  if ! cond_redirect systemctl -q daemon-reload &> /dev/null; then echo "FAILED (daemon-reload)"; return 1; fi
+  if ! cond_redirect zram_dependency install grafana-server; then return 1; fi
```

Comment:
more mixes

Reply:
That had everything to do with CI, I spent quite a bit of time trying to get Grafana to work with the new Initsystem in BAT, and it did on my system, but for whatever reason it would not work properly in the GH Actions env online. That is why the initsystem is only being used on the installation tests and the BATS tests still use the systemctl replacement.